conanfile.py

```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     name = "openh264"
     version = "1.7.0"
     url = "https://github.com/bincrafters/conan-openh264"
+    author = "bincrafters <bincrafters@gmail.com>"
```

Comment:
uppercase `Bincrafters`

Reply:
ok, is this checked in bincrafters-conventions?